src/DevopnessApiClient.ts

```diff
@@ -6,14 +6,22 @@ import { NetworkRuleService } from './services/NetworkRuleService';
 import { ServerService } from './services/ServerService';
 import { ServiceService } from './services/ServiceService';
 import { DaemonService } from './services/DaemonService';
+import { DeploymentsService } from './services/DeploymentsService'
+import { SshKeysService } from './services/SshKeysService';
+import { SslCertificatesService } from './services/SslCertificatesService';
+import { SourceProvidersService } from './services/SourceProvidersService'
 
```

Comment:
Previous service had their name in singular form, new ones are pluralized. It'd be nice to adhere to some naming convention for the services classes. Any preference?

Reply:
oh yeah indeed, let me refactor those to singular